test/stat/stat_test.go

```diff
@@ -96,6 +105,18 @@ func TestCliStatForConduitNamespace(t *testing.T) {
 				TestHelper.GetConduitNamespace(): "4/4",
 			},
 		},
+		{
+			args: []string{"stat", "po", "-n", TestHelper.GetConduitNamespace(), "--to", "au/prometheus.conduit.svc.cluster.local:9090"},
```

Comment:
Unfortunately "au/prometheus.conduit.svc.cluster.local:9090" only works when conduit is running in the default "conduit" namespace, which won't be the case in CI. Instead, I think you should define a separate `prometheusAuthority` var above:

```
	prometheusAuthority := "prometheus." + TestHelper.GetConduitNamespace() + ".svc.cluster.local:9090"
```

And then use that here:

```
			args: []string{"stat", "po", "-n", TestHelper.GetConduitNamespace(), "--to", "au/" + prometheusAuthority},
```